### PR TITLE
Add conversion to YAML

### PR DESCRIPTION
**Pull request by dshevtsov**
The schemas processed by redoc were converted to YAML after 2.4.5 release.
Add such conversion to the automation.

### CODE REVIEW

File: bin/functions.sh [hunk 1]
```diff
@@ -1,3 +1,11 @@
+#!/bin/sh
+
+convert () {
+  ruby -ryaml -rjson -e 'puts JSON.parse(ARGF.read).to_yaml' < __output__/admin-schema-processed-$VERSION.json > __output__/admin-schema-redocly-$VERSION.json
+  ruby -ryaml -rjson -e 'puts JSON.parse(ARGF.read).to_yaml' < __output__/customer-schema-processed-$VERSION.json > __output__/admin-schema-redocly-$VERSION.json
+  ruby -ryaml -rjson -e 'puts JSON.parse(ARGF.read).to_yaml' < __output__/guest-schema-processed-$VERSION.json > __output__/admin-schema-redocly-$VERSION.json
+}
```
Comment (jhadobe) [inline bin/functions.sh]: Need to update the output names for Customer and Guest.

**Review by jhadobe — CHANGES_REQUESTED** [review]
The output names just cause the Admin-Redocly file to be overwritten. Also, they need to have the `.yaml` extension.

Also, I previously noticed the original Admin schema is identical to the original Guest schema. My guess is that the OTP is failing to apply and since the Guest schema is returned by default. This is probably out-of-scope for this PR.

File: bin/functions.sh (hunk 1 above)
Comment (dshevtsov) [reply bin/functions.sh]: Good catch. It's a copy/paste bug.

**Comment by dshevtsov** [comment]
> Also, I previously noticed the original Admin schema is identical to the original Guest schema. My guess is that the OTP is failing to apply and since the Guest schema is returned by default. This is probably out-of-scope for this PR.

That's right. I remember checking the schema during my review and the size was different. I'll re-check.

**Comment by dshevtsov** [comment]
```
➜  commerce-webapi git:(main) ls -lh src/openapi
total 2.5M
-rw-r--r-- 1 dimashevtsov staff 921K Dec 20 13:21 admin-schema-2.3.7.yaml
-rw-r--r-- 1 dimashevtsov staff 964K Mar 17 11:42 admin-schema-2.4.6.yaml
-rw-r--r-- 1 dimashevtsov staff 168K Dec 20 13:21 customer-schema-2.3.7.yaml
-rw-r--r-- 1 dimashevtsov staff 204K Mar 17 11:42 customer-schema-2.4.6.yaml
-rw-r--r-- 1 dimashevtsov staff 130K Dec 20 13:21 guest-schema-2.3.7.yaml
-rw-r--r-- 1 dimashevtsov staff 162K Mar 17 11:42 guest-schema-2.4.6.yaml
```

**Review by jhadobe — APPROVED** [review]
Approving since the YAML conversion works now. The Admin authentication still seems to fail and defaults to the Guest schema

**Comment by dshevtsov** [comment]
> ```
> ➜  commerce-webapi git:(main) ls -lh src/openapi
> total 2.5M
> -rw-r--r-- 1 dimashevtsov staff 921K Dec 20 13:21 admin-schema-2.3.7.yaml
> -rw-r--r-- 1 dimashevtsov staff 964K Mar 17 11:42 admin-schema-2.4.6.yaml
> -rw-r--r-- 1 dimashevtsov staff 168K Dec 20 13:21 customer-schema-2.3.7.yaml
> -rw-r--r-- 1 dimashevtsov staff 204K Mar 17 11:42 customer-schema-2.4.6.yaml
> -rw-r--r-- 1 dimashevtsov staff 130K Dec 20 13:21 guest-schema-2.3.7.yaml
> -rw-r--r-- 1 dimashevtsov staff 162K Mar 17 11:42 guest-schema-2.4.6.yaml
> ```

This is showing different sizes for schemas. What failure do you mean, @jhadobe?

**Comment by jhadobe** [comment]
@dshevtsov Yes, the file sizes are still the same when I run this locally:
file sizes

**Comment by dshevtsov** [comment]
So it had worked when you created the PR, and later it started to fail, right?

**Comment by jhadobe** [comment]
> So it had worked when you created the PR, and later it started to fail, right?

No, every time I've used the transformer, it created `Admin` and `Guest` schemas that were the same size.

**Comment by dshevtsov** [comment]
> ```
> ➜  commerce-webapi git:(main) ls -lh src/openapi
> total 2.5M
> -rw-r--r-- 1 dimashevtsov staff 921K Dec 20 13:21 admin-schema-2.3.7.yaml
> -rw-r--r-- 1 dimashevtsov staff 964K Mar 17 11:42 admin-schema-2.4.6.yaml
> -rw-r--r-- 1 dimashevtsov staff 168K Dec 20 13:21 customer-schema-2.3.7.yaml
> -rw-r--r-- 1 dimashevtsov staff 204K Mar 17 11:42 customer-schema-2.4.6.yaml
> -rw-r--r-- 1 dimashevtsov staff 130K Dec 20 13:21 guest-schema-2.3.7.yaml
> -rw-r--r-- 1 dimashevtsov staff 162K Mar 17 11:42 guest-schema-2.4.6.yaml
> ```

And how did you get these?

**Comment by jhadobe** [comment]
> > ```
> > ➜  commerce-webapi git:(main) ls -lh src/openapi
> > total 2.5M
> > -rw-r--r-- 1 dimashevtsov staff 921K Dec 20 13:21 admin-schema-2.3.7.yaml
> > -rw-r--r-- 1 dimashevtsov staff 964K Mar 17 11:42 admin-schema-2.4.6.yaml
> > -rw-r--r-- 1 dimashevtsov staff 168K Dec 20 13:21 customer-schema-2.3.7.yaml
> > -rw-r--r-- 1 dimashevtsov staff 204K Mar 17 11:42 customer-schema-2.4.6.yaml
> > -rw-r--r-- 1 dimashevtsov staff 130K Dec 20 13:21 guest-schema-2.3.7.yaml
> > -rw-r--r-- 1 dimashevtsov staff 162K Mar 17 11:42 guest-schema-2.4.6.yaml
> > ```
> 
> And how did you get these?

The actual schemas published in the `webapi` repo were from manual API calls.

**Pull request merged**